build.gradle

```diff
@@ -47,13 +58,32 @@ apply from: rootProject.file('gradle/code-quality.gradle')
 
 dependencies {
     compile 'com.gluonhq:charm-down-plugin-storage:3.8.0'
+    compile "org.openjfx:javafx-controls:11-ea+23:$classifier"
+    compile "org.openjfx:javafx-graphics:11-ea+23:$classifier"
+    compile "org.openjfx:javafx-base:11-ea+23:$classifier"
 }
 
+compileJava {
+    doFirst {
+        options.compilerArgs = [
+            '--module-path', classpath.asPath,
```

Comment:
We can update the jdk version that is used in travis as well. See https://github.com/javafxports/openjdk-jfx/blob/develop/.travis.yml#L4

Reply:
we'll have to do that after this is merged, right?

Reply:
I think it should actually be done first. travis build will fail once this PR is merged because java 8 doesn't understand the newly introduced javac arguments.